javascript/javascript1/week3/noteTakingApp.js

```diff
@@ -0,0 +1,38 @@
+const notes = [];
+
+function saveNote(content, id) {
+    if (notes.find(item => item.id === id)) {
+        console.log(`The Id you just entered ${id}, exists in notes please enter another ID`)
+    } else {
+        notes.push({ content: content, id: id })
+    }
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+function getNote(id) {
+    const note = notes.find(item => item.id === id)
+    return note;
+}
+
+const firstNote = getNote(1);
+console.log(firstNote); // {content: 'Pick up groceries', id: 1}
+
+function logOutNotesFormatted() {
+    let allNotes = ""
+    for (let item of notes) {
+        allNotes += `The note with id: ${item.id}, has the following note text: ${item.content}\n`
+    }
+    console.log(allNotes);
+}
+function deleteNote(id) {
+    let noteToDelete = notes.find(item => item.id === id);
+    let indexToDelte = notes.indexOf(noteToDelete);
```

Comment:
Take a look at `findIndex`! Can you think of a way to simplify this code with it? 😄

Reply:
notes.includes(id)?

Reply:
See if you can find the method `findIndex` on arrays. It should help you turn these two lines into one! 🤩